alembic/versions/27a555f0c123_add_create_and_update_timestamps_on_all_.py

```diff
@@ -0,0 +1,78 @@
+"""add create and update timestamps on all records
+
+Revision ID: 27a555f0c123
+Revises: 20dd40a4c970
```

Comment:
@koitoror You will need to update the revision of this alembic file as it is no longer up to date with what we have on develop.

Reply:
Done